DatadogTrace/Sources/OpenTelemetry/OTelSpan.swift

```diff
@@ -86,23 +86,54 @@ internal class OTelSpan: OpenTelemetryApi.Span {
         )
     }
 
-    // swiftlint:disable unavailable_function
+    /// Sends a span event which is akin to a log in Datadog
+    /// - Parameter name: name of the event
     func addEvent(name: String) {
-        fatalError("Not implemented yet")
+        addEvent(name: name, timestamp: .init())
     }
 
+    /// Sends a span event which is akin to a log in Datadog
+    /// - Parameters:
+    ///  - name: name of the event
+    /// - timestamp: timestamp of the event
```

Comment:
nit/ There is inconsistency in indentation of this comment. Shouldn't we also indent parameters list more than the `- Parameters:` header? Same applies to other comments in this file.

Reply:
fixed.